x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/alerting/custom_threshold/consumers_and_privileges.ts

```diff
@@ -107,7 +107,11 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
     });
 
     describe('Custom threshold - Rule visibility - consumer observability', () => {
```

Comment:
`this.tags` doesn't work with arrow style functions, please make it a regular function
```suggestion
    describe('Custom threshold - Rule visibility - consumer observability', function () {
```